.github/workflows/main.yml

```diff
@@ -19,7 +19,16 @@ jobs:
         run: |
           python -m pip install tox
       - name: Run tests
-        run: tox
+        run: tox -- --junit-xml=.test-results/pytest/cov.xml
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          file: .test-results/pytest/cov.xml  # Explicitly point at the correct coverage report
+          flags: >-  # Mark which lines are covered by which envs
+            GHA,
```

Comment:
Since there are no environments that don't use Github actions, it's probably premature to mark these tests as GHA. Probably better to eliminate it until it's needed.

Reply:
Fixed in a77319fff.